service/src/main/java/aero/minova/cas/SecurityConfig.java

```diff
@@ -19,22 +22,26 @@
 import org.springframework.security.ldap.authentication.ad.ActiveDirectoryLdapAuthenticationProvider;
 import org.springframework.security.ldap.userdetails.LdapUserDetailsMapper;
 import org.springframework.security.ldap.userdetails.UserDetailsContextMapper;
+import org.springframework.web.cors.CorsConfiguration;
+import org.springframework.web.cors.CorsConfigurationSource;
+import org.springframework.web.cors.UrlBasedCorsConfigurationSource;
 import org.thymeleaf.extras.springsecurity4.dialect.SpringSecurityDialect;
 
 import aero.minova.cas.service.SecurityService;
 import aero.minova.cas.sql.SystemDatabase;
 
 @EnableWebSecurity
 @Configuration
-public class SecurityConfig extends WebSecurityConfigurerAdapter {
+public class SecurityConfig {
```

Comment:
Die Struktur sollte eher folgende sein:

```
@Configuration
public class SecurityConfig {
    @EnableWebSecurity
    @Order(1)
    @Configuration
    public static class BaseSecurityConfig extends WebSecurityConfigurerAdapter {
        [...]
    }
    @Order(2)
    @Configuration
    @Profile("dev")
    public static class DevSecurityConfig extends WebSecurityConfigurerAdapter {
        [...]
    }
}
```

Reply:
Das hatten wir tatsächlich auch so, aber laut folgender Anleitung ist es richtig und wichtig, die generische - also StandardConfig - an letzter Stelle zu setzen:

"Make sure to put the most generic configuration as the last order" 
( siehe https://medium.com/@igor.bonny/multiple-spring-boot-security-configuration-c876f1b6061e )

Unsere Tests haben dies leider auch bestätigt.